lbrynet/daemon/Daemon.py

```diff
@@ -997,17 +997,16 @@ def _blob_availability(self, blob_hash, search_timeout, blob_timeout, downloader
     ############################################################################
 
     @defer.inlineCallbacks
-    @AuthJSONRPCServer.flags(session_status="-s", dht_status="-d")
```

Comment:
the `flags` decorator can be removed from `AuthJSONRPCServer`

Reply:
From here?
https://github.com/lbryio/lbry/blob/master/lbrynet/daemon/auth/server.py#L153